docs/source/nn.attention.rst

```diff
@@ -17,12 +17,15 @@ Utils
 
 Submodules
 ----------
+
 .. autosummary::
     :nosignatures:
 
+    templated_attention
     bias
 
 .. toctree::
     :hidden:
 
-    nn.attention.bias
\ No newline at end of file
+    nn.attention.templated_attention
```

Comment:
So if we want to keep the higher level api private we can do that but we need to name it `_templated_attention`.  Otherwise we now test that every torch/*.py module has docs.

I am fine either way